docs/operations/metrics.md

```diff
@@ -298,6 +298,11 @@ If the JVM does not support CPU time measurement for the current thread, `ingest
 |`worker/task/assigned/count`|Number of tasks assigned to an indexer per emission period. This metric is only available if the `WorkerTaskCountStatsMonitor` module is included.|`dataSource`|Varies|
 |`worker/task/completed/count`|Number of tasks completed by an indexer per emission period. This metric is only available if the `WorkerTaskCountStatsMonitor` module is included.|`dataSource`|Varies|
 |`worker/task/running/count`|Number of tasks running on an indexer per emission period. This metric is only available if the `WorkerTaskCountStatsMonitor` module is included.|`dataSource`|Varies|
+|`s3upload/chunk/queueSize`|The number of tasks that are currently queued and waiting to upload a part to S3 (in a multi-part upload) when durable storage is enabled.||Varies|
+|`s3upload/chunk/queuedTime`|Milliseconds spent by a task in queue before it starts uploading a part to S3 (in a multi-part upload) when durable storage is enabled.|`uploadId`, `partNumber`|Varies|
+|`s3upload/chunk/uploadTime`|The time taken in milliseconds to upload a part to S3 (in a multi-part upload) when durable storage is enabled.|`uploadId`, `partNumber`|Varies|
+|`s3upload/job/totalTime`|The total time taken in milliseconds for uploading all parts of a file to S3 when durable storage is enabled.|`uploadId`|Varies|
+|`s3upload/job/totalBytes`|The total number of bytes uploaded across all parts of a file to S3 when durable storage is enabled.|`uploadId`|Varies|
```

Comment:
@Akshat-Jain , these metrics will only be emitted if S3 extensions are loaded and if certain specific features are being used i.e. MSQ export to S3 or durable intermediate storage on S3.

As such, these metrics should be in a separate section, similar to say [`OshiSysMonitor`](https://druid.apache.org/docs/latest/operations/metrics/#oshisysmonitor).

Keeping that in mind, better names for these metrics would be:
```
s3/upload/part/time
s3/upload/part/queuedTime
s3/upload/part/queueSize

s3/upload/time (or s3/upload/total/time)
s3/upload/bytes (or s3/upload/total/bytes)
```

I think we should use `part` instead of `chunk` as multi-part upload is the known name of this feature.

Reply:
Made the changes, thanks!